src/vue/common/TabsPanel.vue

```diff
@@ -0,0 +1,75 @@
+<template>
+  <div class="tabs-panel">
+    <div class="tabs-panel__main">
+      <slot />
```

Comment:
the component actually doesn’t accept anything that looks like "tabs", also maybe it makes sense to rename the component to better describe the role it plays. "tabs-component" sounds not such global as it have to be, something describing it role "position on top panel" would be great.

Reply:
Resolved, updated